src/components/InviteFriends/index.tsx

```diff
@@ -0,0 +1,220 @@
+      headers: {
+        "Content-Type": "application/x-www-form-urlencoded",
+      },
+    });
+    const portKeyResObj = await portKeyRes.json();
+    if (portKeyRes.ok) {
+      const token = portKeyResObj.access_token;
+      const res = await fetch(
+        portkeyServer + "/api/app/growth/shortLink?projectCode=13027",
+        {
+          method: "GET",
+          headers: {
+            Authorization: `Bearer ${token}`,
+          },
+        }
+      );
+      const resObj = await res.json();
+      if (res.ok) {
+        return resObj as ShortLinkRes;
+      }
+    }
+    return null;
+  };
+
+  const handleCopy = () => {
+    setCopied(tgLinkWithCode);
+    setIsCopied(true);
+  };
+
+  const { data: referralCodeRes, loading } = useRequest(async () => {
+    return generateCode();
+  });
+
+  const inviteCode = referralCodeRes?.userGrowthInfo?.inviteCode;
+  const startAppParams: IStartAppParams = {
+    referralCode: inviteCode,
+  };
+  const tgLinkWithCode =
+    TgLink +
+    (inviteCode ? `?startapp=${stringifyStartAppParams(startAppParams)}` : "");
+
+  useEffect(() => {
+    if (isCopied) {
+      setTimeout(() => {
+        setIsCopied(false);
+      }, 2000);
+    }
+  }, [isCopied]);
+
+  return (
+    <>
+      <div className="p-[1px] rounded-[18px] w-full !content-[''] bg-gradient-to-r from-lime-primary to-lime-green">
+        <div className="px-5 pt-[22px] pb-4 bg-black rounded-[18px]">
+          <div className="flex items-start justify-between mb-[14px]">
+            <span className="whitespace-pre-wrap -tracking-[0.5px] font-bold font-outfit text-[28px] leading-[28px] text-white">
+              {"Invite friends\n& get points!"}
+            </span>
+            <span className="mt-2 text-lg font-normal text-[13px] leading-[15.6px] text-lime-green">
+              +{data?.pointsFirstReferralVote?.toLocaleString()}
+            </span>
+          </div>
+          <div className="flex items-center justify-between mb-[14px] gap-5">
+            <div className="flex-1 h-[10px] shrink-0 bg-white rounded-[5px]">
+              <div
+                className={clsx(
+                  "h-[10px] bg-gradient-to-r from-lime-green to-lime-primary rounded-[5px]"
+                )}
+                style={{ width: `${progress}%` }}
+              />
+            </div>
+            <span className="font-outfit font-bold text-[14px] leading-[14px] text-white shrink-0">
+              {data?.votigramVoteAll || 0}/{data?.totalInvitesNeeded || 20}
+            </span>
+          </div>
+          <button
+            type="button"
+            onClick={onClickInvite}
+            className="w-full  bg-primary rounded-[24px] text-[14px] font-bold py-[10px] font-outfit leading-[14px] text-white"
+          >
+            Invite friends
+          </button>
+        </div>
+      </div>
+      <Drawer
+        isVisible={isShowDrawer}
+        direction="bottom"
+        canClose={true}
```

Comment:
if its true, you do not need to pass true

Reply:
Updated